.rhiza/completions/rhiza-completion.bash

```
@@ -0,0 +1,47 @@
+#   Source this file in your ~/.bashrc or ~/.bash_profile:
+#     source /path/to/.rhiza/completions/rhiza-completion.bash
+#
+#   Or copy to bash completion directory:
+#     sudo cp .rhiza/completions/rhiza-completion.bash /etc/bash_completion.d/rhiza
+#
+
+_rhiza_make_completion() {
+    local cur prev opts
+    COMPREPLY=()
+    cur="${COMP_WORDS[COMP_CWORD]}"
+    prev="${COMP_WORDS[COMP_CWORD-1]}"
```

Shellcheck (0.11.0)

[warning] prev appears unused. Verify use (or export if used externally).

(SC2034)